interop-tests/README.md

```diff
@@ -8,13 +8,11 @@ You can run this test locally by having a local Redis instance and by having
 another peer that this test can dial or listen for. For example to test that we
 can dial/listen for ourselves we can do the following:
 
-1. Start redis (needed by the tests): `docker run --rm -it -p 6379:6379
-   redis/redis-stack`.
+1. Start redis (needed by the tests): `docker run --rm -p 6379:6379 redis:7-alpine`.
```

Comment:
What is the motivation behind changing the redis image?

Reply:
When I tried the original redis command it didn't work, then I cross referenced the command in the test plans (https://github.com/libp2p/test-plans/tree/master/multidim-interop#running-locally) which did work. So I made the change to make them consistent with the command that worked